tsfeatures/tests/test_pacf_features.py

```diff
@@ -1,10 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Please add these tests to nbs/tsfeatures.ipynb

Reply:
added the tests to the tsfeatures.ipynb